### PR TITLE
adding the Solidity Survey

### PR DESCRIPTION
**Pull request by LianaHus**

### CODE REVIEW

File: libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx [hunk 1]
```diff
@@ -40,6 +40,38 @@ function HomeTabFeatured() {
...
+              <div className="mx-1 px-1 d-flex">
+                <a href="https://cryptpad.fr/form/#/2/form/view/pV-DdryeJoYUWvW+gXsFaMNynEY7t5mUsgeD1urgwSE/" target="__blank">
+                  <img src={'/assets/img/soliditySurvey2023.webp'} style={{flex: '1', height: '170px', maxWidth: '170px'}} alt=""></img>
+                </a>
+                <div className="h6 w-50 p-2 pl-4  align-self-center" style={{flex: '1'}}>
+                  <h5>
+                  The Solidity Developer Survey 2023 is live!
+                  </h5>
+                  <p className='pt-2'>
+                    Please take a few minute of your time to
```
Comment (Aniket-Engg) [inline libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx:52]: minute => minutes

File: libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx [hunk 2]
```diff
@@ -40,6 +40,38 @@ function HomeTabFeatured() {
...
+                  </h5>
+                  <p className='pt-2'>
+                    Please take a few minute of your time to 
+                    <a
+                      className="mx-1"
+                      onClick={() => _paq.push(['trackEvent', 'hometab', 'featuredSection', 'soliditySurvey23'])}
+                      target="__blank"
+                      href="https://soliditylang.org/blog/2023/12/08/solidity-developer-survey-2023-announcement/"
+                    >
+                      COMPLETE THE SURVEY.
```
Comment (Aniket-Engg) [inline libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx:59]: Better to add this on button

**Review by Aniket-Engg — COMMENTED** [review]
There are 3 different font sizes used for text. That looks quite asymmetric. I would like to suggest to use 2 font sizes and to keep this slide consistent with others.

File: libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx (hunk 2 above)
Comment (LianaHus) [reply libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx:59]: Doesn't look good on a button. It was a btn on the first iteration I had to change it

File: libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx [hunk 3]
```diff
@@ -40,7 +40,39 @@ function HomeTabFeatured() {
...
+                  </p>
+                  <p className="mb-3">
+                    Thank you for your support! Read the full announcement
+                    <a
+                      className="remixui_home_text mx-1"
+                      onClick={() => _paq.push(['trackEvent', 'hometab', 'featuredSection', 'soliditySurvey23'])}
+                      target="__blank"
+                      href="https://soliditylang.org/blog/2023/12/08/solidity-developer-survey-2023-announcement/"
+                    >
+                      here.
```
Comment (Aniket-Engg) [inline libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx:62]: `here` text is small comparing to complete line

Comment (LianaHus) [reply libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx:62]: it's a link

**Review by Aniket-Engg — APPROVED** [review]

**Pull request merged**